### PR TITLE
Retrieve cluster_id from application credentials

### PR DESCRIPTION
**Pull request by pierreprinetti**

### CODE REVIEW

**Comment by pierreprinetti** [comment]
/hold cancel
https://github.com/openshift/release/pull/43348 worked.

File: applicationcredentials.go [hunk 1]
```diff
@@ -35,6 +36,16 @@ func (s ApplicationCredential) Name() string {
 	return s.resource.Name
 }
 
+func (s ApplicationCredential) ClusterID() string {
+	for _, tag := range strings.Split(s.resource.Description, " ") {
+		// https://github.com/openshift/release/pull/43348
```
Comment (mandre) [inline applicationcredentials.go:41]: We can remove the link now that the dependent patch has merged, WDYT?

Comment (pierreprinetti) [reply applicationcredentials.go:41]: I like that we keep a trace of why this works. There is the same for [images](https://github.com/shiftstack/prune/blob/08b0af97ae0dcb693d73d0fe8733b645e3e7cd01/images.go#L39) and [floating IPs](https://github.com/shiftstack/prune/blob/08b0af97ae0dcb693d73d0fe8733b645e3e7cd01/floatingips.go#L42)

Comment (mandre) [reply applicationcredentials.go:41]: Fine.

**Review by mandre — COMMENTED** [review]
/lgtm

**Pull request merged**